vortex-datafusion/src/persistent/sink.rs

```diff
@@ -385,7 +385,7 @@ mod tests {
         register_vortex_format_factory(factory, &mut session_state_builder);
         let session = SessionContext::new_with_state(session_state_builder.build());
 
-        let _ = session
+        let _unused = session
```

Comment:
This is a bit suspicious to me? Shouldn't we fail if the external table fails to create?

Reply:
isn't that handled by the `.await?` ?